integration_tests/src/main/python/udf_test.py

```diff
@@ -425,3 +427,35 @@ def test_func(spark):
             lambda data: [pd.DataFrame([len(list(data))])], schema="ret:integer")
 
     assert_gpu_and_cpu_are_equal_collect(test_func, conf=arrow_udf_conf)
+
+
+@pytest.mark.skipif(is_before_spark_350(),
+                    reason='mapInPandas/mapInArrow with barrier mode is introduced by Pyspark 3.5.0')
+@pytest.mark.parametrize('is_map_in_pandas', [True, False], ids=idfn)
```

Comment:
actually, i mean comparison between CPU and GPU results for both barrier and non-barrier modes, not the GPU results for the two modes.

Reply:
Done. Thx